projects/arc-docs/src/app/docs/auth-doc/components/configure-token-doc/configure-token-doc.component.html

```diff
@@ -0,0 +1,67 @@
+<div class="intro-wrapper">
+  <div class="1st-phase">
+    <nb-card>
+      <nb-card-header fixed>
+        <h2 class="nb-card-header-title">
+          Receiving user token after login/registration
+        </h2>
+      </nb-card-header>
+      <nb-card-body>
+        <ul>
+          <li>
+            At this step,It's time to get a user token but primarly we need a
+            code with the help of code we call getToken API which returns a
+            token, after a successful authentication to be able to communicate
+            with the server and, for instance, show username in the header of
+            the application
+          </li>
+          <li>
+            Let's assume that your backend returns a JWT token so that we can
+            use the token payload to extract user info out of it. Each Strategy
+            specifies which token class is going to be used by default.
+          </li>
+          <li>
+            For example, NbPasswordAuthStrategy uses AuthToken, and AuthProvider
+            . It is also possible to specify another token class if it is
+            required.
+          </li>
+        </ul>
+      </nb-card-body>
+    </nb-card>
+
+    <nb-card>
+      <nb-card-header fixed>
+        <h2 class="nb-card-header-title">Setting a User token</h2>
+      </nb-card-header>
+      <nb-card-body>
+        <p>
+          When we get token we call authorise method, The authorize method is
+          part of an authentication service. It is designed to handle the
+          authorization process using a provided secret (such as an
+          authorization code). This method initiates a command to obtain tokens,
+          processes the response, and navigates the user based on the
+          authorization result.
```

Comment:
not making sense and this is not what we do

Reply:
ok